CMakeLists.txt

```diff
@@ -122,11 +123,13 @@ if(FCL_WITH_OCTOMAP)
           set(OCTOMAP_LIBRARIES "octomap;octomath")
       endif()
   endif()
-  if (OCTOMAP_FOUND OR (OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS))
-    string(REPLACE "." ";" VERSION_LIST ${OCTOMAP_VERSION})
-    list(GET VERSION_LIST 0 OCTOMAP_MAJOR_VERSION)
-    list(GET VERSION_LIST 1 OCTOMAP_MINOR_VERSION)
-    list(GET VERSION_LIST 2 OCTOMAP_PATCH_VERSION)
+  if(OCTOMAP_INCLUDE_DIRS AND OCTOMAP_LIBRARY_DIRS AND OCTOMAP_VERSION)
+    if(NOT OCTOMAP_MAJOR_VERSION AND NOT OCTOMAP_MINOR_VERSION AND NOT OCTOMAP_PATCH_VERSION)
+      string(REPLACE "." ";" VERSION_LIST "${OCTOMAP_VERSION}")
```

Comment:
My concern was the case that octomap is installed and found by `find_package`, but FCL doesn't use it only because `OCTOMAP_VERSION` is not defined (due to `octomap-config.cmake` doesn't define it).